content/rs/references/cli-utilities/rladmin/info.md

```diff
@@ -0,0 +1,193 @@
+    master_persistence: disabled
+    maxclients: 10000
+    conns: 5
+    conns_type: per-thread
+    sched_policy: cmp
+    max_aof_file_size: 300GB
+    max_aof_load_time: 3600 seconds
+    dedicated_replicaof_threads: 5
+    max_client_pipeline: 200
+    max_shard_pipeline: 2000
+    max_connections: 0
+    oss_cluster: disabled
+    oss_cluster_api_preferred_ip_type: internal
+    gradual_src_mode: disabled
+    gradual_src_max_sources: 1
+    gradual_sync_mode: auto
+    gradual_sync_max_shards_per_source: 1
+    slave_ha: disabled (database)
+    mkms: enabled
+    oss_sharding: disabled
+    mtls_allow_weak_hashing: disabled
+    mtls_allow_outdated_certs: disabled
+    data_internode_encryption: disabled
+    proxy_policy: single
+    db_conns_auditing: disabled
+```
+
+## `info proxy`
+
+`rladmin info proxy` lists configurable settings for the proxy.
+
+```sh
+rladmin info proxy
+```
+
+### Parameters
+
+None.
+
+### Returns
+
+Returns a list of configurable settings for the proxy.
+
+### Example
+
+``` sh
+$ rladmin info proxy
+proxy:1
+    mode: dynamic
+    scale_threshold: 80 (%)
+    scale_duration: 30 (seconds)
+    max_threads: 8
+    threads: 3
+```
+
+## `info cluster`
+
+`rladmin info cluster` lists configurable settings for the cluster.
+
+```sh
+rladmin info cluster
+```
+
+### Parameters
+
+None.
+
+### Returns
+
+Returns a list of configurable settings for the cluster.
+
+### Example
+
+``` sh
+$ rladmin info cluster
+Cluster configuration:
+    repl_diskless: enabled
+    shards_overbooking: disabled
+    default_non_sharded_proxy_policy: single
+    default_sharded_proxy_policy: single
+    default_shards_placement: dense
+    default_fork_evict_ram: enabled
+    default_provisioned_redis_version: 6.0
+    redis_migrate_node_threshold: 0KB (0 bytes)
+    redis_migrate_node_threshold_percent: 4 (%)
+    redis_provision_node_threshold: 0KB (0 bytes)
+    redis_provision_node_threshold_percent: 12 (%)
+    max_simultaneous_backups: 4
+    slave_ha: enabled
+    slave_ha_grace_period: 600
+    slave_ha_cooldown_period: 3600
+    slave_ha_bdb_cooldown_period: 7200
+    parallel_shards_upgrade: 0
+    show_internals: disabled
+    expose_hostnames_for_all_suffixes: disabled
+    login_lockout_threshold: 5
+    login_lockout_duration: 1800
+    login_lockout_counter_reset_after: 900
+    default_concurrent_restore_actions: 10
+    endpoint_rebind_propagation_grace_time: 15
+    data_internode_encryption: disabled
+    redis_upgrade_policy: major
+    db_conns_auditing: disabled
+    watchdog profile: local-network
+    http support: enabled
+    upgrade mode: disabled
+    cm_session_timeout_minutes: 15
+    cm_port: 8443
+    cnm_http_port: 8080
+    cnm_https_port: 9443
+```
+
+## `info node`
+
+`rladmin info node` lists configurable settings for all nodes.
+
+```sh
+rladmin info db [ <id> ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| id     | node id of the specified node |
```

Comment:
```suggestion
|  id  | ID of the specified node  |
```

Reply:
Fixed in [217ff69](https://github.com/RedisLabs/redislabs-docs/pull/1887/commits/217ff6923523a42662d82993b676c3c72d98cb9c)